AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift

```diff
@@ -11,46 +11,29 @@ import Foundation
 /// `AudioVideoConfiguration` represents the configuration to be used for audio and video during a meeting session.
 @objcMembers public class AudioVideoConfiguration: NSObject {
     public let audioMode: AudioMode
+    public let audioDeviceCapabilities: AudioDeviceCapabilities
     public let callKitEnabled: Bool
     public let enableAudioRedundancy: Bool
     public let videoMaxResolution: VideoResolution
 
-    convenience override public init() {
-        self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
-    }
-
-    convenience public init(audioMode: AudioMode) {
-        self.init(audioMode: audioMode, callKitEnabled: false, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
-    }
-
-    convenience public init(callKitEnabled: Bool) {
-        self.init(audioMode: .stereo48K, callKitEnabled: callKitEnabled, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
-    }
-
-    convenience public init(enableAudioRedundancy: Bool) {
-         self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: enableAudioRedundancy, videoMaxResolution: VideoResolution.videoResolutionHD)
-    }
-
-    convenience public init(videoMaxResolution: VideoResolution) {
-        self.init(audioMode: .stereo48K, callKitEnabled: false, enableAudioRedundancy: true, videoMaxResolution: videoMaxResolution)
-    }
-
-    convenience public init(audioMode: AudioMode, callKitEnabled: Bool) {
-        self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: true, videoMaxResolution: VideoResolution.videoResolutionHD)
-    }
-
-    convenience public init(audioMode: AudioMode, callKitEnabled: Bool, enableAudioRedundancy: Bool) {
-        self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: enableAudioRedundancy, videoMaxResolution: VideoResolution.videoResolutionHD)
-    }
-
-    public init(audioMode: AudioMode, callKitEnabled: Bool, enableAudioRedundancy: Bool, videoMaxResolution: VideoResolution) {
+    public init(audioMode: AudioMode = .stereo48K,
```

Comment:
I think those convenience are there for obj-c support. Have you verified this new init with default values can service the same purpose?

Reply:
Ah I didn't realize that it was for obj-c, i'll verify and add back those convenience methods if the obj-c code doesn't work without the convenience methods

Reply:
Added back the convenience inits because they are still needed for obj-c